docs/developers/index.rst

```diff
@@ -0,0 +1,12 @@
+=========================
+Developer's Documentation
```

Comment:
breaking a doc set up at the top level, for the same project, into a user part and a developer part is not recommended

i suggest having the doc set be end-user by default, so no change needed there, but adding a high-level section for developers:

"Developer documentation"
"For developers"
"Contributing to MicroOVN"

Reply:
Ack. I wanted to avoid asymmetry of layout:

* How-tos
* Tutorials
* For developers
  * How-tos
  * Tutorials

I though that branching on the top would be better

Reply:
the [Snap documentation](https://snapcraft.io/docs) is in a similar situation. There are snap users and snap builders.

Reply:
maybe we should not factor into Diataxis yet (we can do that when warranted (more pages) - just place them directly under the new heading for now